release/_releases_json.sh

```diff
@@ -1,6 +1,31 @@
 #!/bin/bash
 
+function _add_new_version {
+	if [ -z "${LIFERAY_RELEASE_PRODUCT_NAME}" ] || [ -z "${LIFERAY_RELEASE_VERSION}" ]
+	then
+		return
+	fi
+
+	if [[ "${LIFERAY_RELEASE_PRODUCT_NAME}" =~ ^(dxp|portal)$ ]] && [[ "${LIFERAY_RELEASE_VERSION}" =~ ^[0-9]+.([0-9])|(q[0-9])$ ]]
+	then
+		if grep -q "${LIFERAY_RELEASE_VERSION}" "${_RELEASE_ROOT_DIR}"/supported-"${LIFERAY_RELEASE_PRODUCT_NAME}"-versions.txt
+		then
+			lc_log INFO "Version is already on the list"
+		else
+			lc_log INFO "Add ${LIFERAY_RELEASE_VERSION} version to supported-${LIFERAY_RELEASE_PRODUCT_NAME}-versions.txt"
+
+			echo -en "\n${LIFERAY_RELEASE_VERSION}" >> "${_RELEASE_ROOT_DIR}"/supported-"${LIFERAY_RELEASE_PRODUCT_NAME}"-versions.txt
+		fi
+	else
+		lc_log INFO "The given values are invalid"
+
+		print_help
+	fi
+}
+
 function regenerate_releases_json {
+	_add_new_version
```

Comment:
I don't think `check `is a good fit for the name because the function is for adding a version after checking if it is already there, `add_supported_version` would be a better option if want to change it I think.